creator-node/src/logging.js

```diff
@@ -26,8 +26,18 @@ function loggingMiddleware (req, res, next) {
     requestID: requestID,
     requestMethod: req.method,
     requestHostname: req.hostname,
-    requestUrl: req.originalUrl
+    requestUrl: req.originalUrl,
+    requestWallet: req.get('user-wallet-addr') ? req.get('user-wallet-addr') : null,
```

Comment:
Just wondering - what's the point of doing a ternary check here if we're just setting it to `null`? Are we just trying to avoid passing in `undefined`?

Reply:
you're right, i guess we don't need the ternary check anymore. i didn't know if i wanted to put some other value here so that's why i did ternary but i can remove now